src/app/dashboard/helper.ts

```diff
@@ -0,0 +1,71 @@
+import { Redis } from "@upstash/redis"
+
+// Initialize Redis outside of the job function
+const redis = new Redis({
+  url: process.env.REDIS_URL,
+  token: process.env.REDIS_TOKEN
+})
+
+interface InviteCodeReturnJsonSuccess {
+  isInviteCodeGenerated: boolean
+  inviteCode: string
+}
+
+type InviteCodeReturnJsonFailure = Omit<InviteCodeReturnJsonSuccess, "inviteCode">
+
+export async function generateInviteCode(
+  role: string
+): Promise<InviteCodeReturnJsonSuccess | InviteCodeReturnJsonFailure> {
+  const rString = "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
+  let inviteCode = ""
+  for (var i = 0; i < 5; i++) {
+    inviteCode += rString[Math.floor(Math.random() * rString.length)]
```

Comment:
Does this always always pick an index less than the length of the string?

Reply:
Think so, I tried generating around 10 pairs of random strings and never found a badly formed string or an error.